ci_framework/roles/rhol_crc/tasks/dependencies.yml

```diff
@@ -0,0 +1,21 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Install packages required for using RHOL/CRC
+  ansible.builtin.package:
+    name: "{{ cifmw_rhol_crc_dependency_packages }}"
```

Comment:
Following what we openstack-ansible do, I would add this in the defaults as a comment, with an example, at least people would know what this is